INSTALL.txt

```diff
@@ -200,8 +200,8 @@ will be just skipped.
 
 6. Run the build script for current stable release:
 
-python ./build_prod.py --build-type dailybuild --machine MACHINE_NAME --product PRODUCT_NAME \
---branch RELEASE_NAME --with-local-conf --config xt-prod-devel.cfg --with-do-build
+python ./build_prod.py --build-type dailybuild --machine MACHINE_NAME --product devel \
+--branch REL-v5.2 --with-local-conf --config xt-prod-devel.cfg --with-do-build
```

Comment:
Not sure this is correct. The documentation says that you may build devel-src as well and implies PRODUCT_NAME variable substitution by the user. The same should be true for RELEASE_NAME as well.

Reply:
here i consider the sample to build REL-v5.2
regarding the prod-devel-src - yes, the sample has to be described